src/test/java/essenmakanan/parser/AddRecipeCommandTest.java

```diff
@@ -182,5 +182,15 @@ public void addValidCommand_stepAndIngredientNotInOrder_recipeCreated() {
         assertEquals(IngredientUnit.PIECE, recipeIngredients.getIngredients().get(0).getUnit());
     }
 
+    @Test
+    public void addRecipeCommand_multipleTitle_formatException() {
```

Comment:
```suggestion
    public void addRecipeCommand_multipleTitles_formatException() {
```
I suggest that the method name should be this.